public/pages/ThreatIntel/containers/AddThreatIntelSource/AddThreatIntelSource.tsx

```diff
@@ -82,13 +132,22 @@ export const AddThreatIntelSource: React.FC<AddThreatIntelSourceProps> = ({
     ]);
   }, []);
 
+  const validateIocTypes = (iocTypeMap: Record<string, boolean>) => {
+    return !Object.values(iocTypeMap).some((val) => val)
+      ? 'At least one ioc type should be selected'
```

Comment:
Not blocking:
Should "ioc" be "IoC", "IOC", or whichever capitalization pattern is consistent with the rest of the UI? Also, I think we want to end error messages with a period.

Reply:
lot's of places, let me do this in a follow up PR